recipes/mmtf-pyspark/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: mmtfpyspark
+  version: "0.2.0"
```

Comment:
It looks like the newest version is `0.2.6`. Unless there is a compelling reason to start with an older version, I would update to the latest version.